lib/client.js

```diff
@@ -41,10 +41,8 @@ var SCROLLBACK_DELAY_MS = 3000;
 var CRYPTO_ENABLED = false;
 
 try {
-    var Olm = require("olm");
-    if (Olm.Account && Olm.Session) {
-        CRYPTO_ENABLED = true;
-    }
+    var OlmDevice = require("./OlmDevice");
+    CRYPTO_ENABLED = true;
```

Comment:
Well, I suppose so. But we don't actually do that (we use the IgnorePlugin, which just turns the require line into a `throw`).